internals/placementController/adapters/capp.go

```diff
@@ -0,0 +1,44 @@
+package adapters
+
+import (
+	"context"
+	"fmt"
+	rcsv1alpha1 "github.com/dana-team/container-app-operator/api/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	// AnnotationKeyHasPlacement is the key used to store the managed cluster name in an annotation on a Capp resource
+	AnnotationKeyHasPlacement = "dana.io/has-placement"
+	// RcsConfigName is the name of the RCS Deployer Config CRD instance
+	RcsConfigName = "rcs-config"
+	// RcsConfigNamespace is the namespace that contains the RCS Deployer Config CRD instance
+	RcsConfigNamespace = "rcs-deployer-system"
```

Comment:
This is going to be deprecated anyways but since RCS is an initialism, this needs to read `RCSConfigName` and `RCSConfigNamespace`